snuba/api.py

```diff
@@ -257,58 +299,11 @@ def parse_and_run_query(dataset, validated_body, timer):
         ], key=lambda priority_and_col: priority_and_col[0])
         if prewhere_candidates:
             prewhere_conditions = [cond for _, cond in prewhere_candidates][:settings.MAX_PREWHERE_CONDITIONS]
-            where_conditions = list(filter(lambda cond: cond not in prewhere_conditions, where_conditions))
-
-    where_clause = ''
-    if where_conditions:
-        where_conditions = util.tuplify(where_conditions)
-        where_clause = u'WHERE {}'.format(util.conditions_expr(dataset, where_conditions, body))
-
-    prewhere_clause = ''
-    if prewhere_conditions:
-        prewhere_clause = u'PREWHERE {}'.format(util.conditions_expr(dataset, prewhere_conditions, body))
+            body['conditions'] = list(filter(lambda cond: cond not in prewhere_conditions, body['conditions']))
```

Comment:
This will probably stay shared but `PREWHERE_KEYS` will become a dataset configuration.